fs/fs_zenfs.cc

```diff
@@ -245,9 +245,9 @@ void ZenFS::LogFiles() {
 }
 
 void ZenFS::ClearFiles() {
-  std::map<std::string, ZoneFile*>::iterator it;
+  std::map<std::string, std::shared_ptr<ZoneFile>>::iterator it;
   files_mtx_.lock();
-  for (it = files_.begin(); it != files_.end(); it++) delete it->second;
+  for (it = files_.begin(); it != files_.end(); it++) it->second.reset();
```

Comment:
This line is redundant. All objects in the map are destroyed when we call `files_.clear()` on the next line.